### DISCUSSION TITLE
edit tools in edit mode

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#16353] [Ideas] [answered]
it would be really helpful if there's edit tools like [mapping tools](https://mappingtools.github.io/)

**Comment by bdach** [comment]
wasn't aware of the existence of this tool. would need to review its many features and check what is indeed helpful and what may be already obsoleted by new lazer-specific features. but definitely no reason not to include some of these functionalities if there is demand for it.

**Comment by peppy** [comment, marked as answer]
A lot of the features provided by external tools already have open discussion or are implemented. Please open individual discussions for things which are missing that you would like to see.

Making a post like this is not too helpful to us, especially when there's no way to know which pieces of an extneral application are actually useful to the bulk of mappers.
